working/augmentation-libraries/feature-specification.md

```diff
@@ -938,79 +756,31 @@ It is a compile-time error if:
     *Default values are defined solely by the introductory constructor.*
```

Comment:
I can't comment on line 751, but 'same type,' should probably be 'same type (when provided),'.

Reply:
Good catch. Done.